src/sincere_singularities/modules/order_queue.py

```diff
@@ -47,9 +47,10 @@ def __init__(self, interaction: ApplicationCommandInteraction, webhook: Webhook)
         self.interaction = interaction
         self.user = interaction.user
         self.orders: dict[str, tuple[Order, WebhookMessage]] = {}
+        self.number_of_orders = 0
```

Comment:
We should save this in the Database.
Is our current temp database good enough to already implement this or should we just wait for @WassCodeur implementation?